app/controllers/spree/admin/taxjar_settings_controller.rb

```diff
@@ -2,7 +2,20 @@ module Spree
   module Admin
     class TaxjarSettingsController < Spree::Admin::BaseController
       def edit
+        @settings = SuperGood::SolidusTaxjar::Settings.find_or_create_by(id: 1)
       end
+
+      def update
+        settings = SuperGood::SolidusTaxjar::Settings.find_or_create_by(id: 1)
+        settings.update!(settings_params)
+        flash[:alert] = "TaxJar settings updated!"
```

Comment:
Is alert a red flash message or green flash message? I didn't see it in the video demo, and I'm used to alerts being red, but this one should probably be green!

Reply:
I think alert is the green one, and error is the red one, but I could be wrong.

Reply:
OK, I was wrong, it's actually `:success` we should be using based on the style guide here https://github.com/solidusio/solidus/blob/master/backend/app/views/spree/admin/style_guide/topics/messaging/_flashes.html.erb
0